README.md

```diff
@@ -103,6 +103,15 @@ $xmlPackage->send();
 // phpcs:enable
 ```
 
+Ignore a specific violation:
+```php
+$xmlPackage = new XMLPackage;
+$xmlPackage['error_code'] = get_default_error_code_value();
+// phpcs:ignore Generic.Commenting.Todo.Found
+// TODO: Add an error message here.
+$xmlPackage->send();
+```
+
```

Comment:
I guess you meant to move disabling the rule? Either change the block above or remove the block above. Now it's double.

Reply:
It's a similar example but a different way to solve it. `phpcs:ignore` is different from `phpcs:disable`, it only ignores the violation of the current or next line.

Reply:
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file mentions and give some examples on it. Tomorrow I'll amend the commit message to add the reference

Reply:
Ah ok, got it. I think it helps when mentioning it ignores the violation for the next line only.